From 1715a389949a4a3f03192268dfb3ac5710f2dd3c Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Mon, 30 Jun 2014 23:57:54 +0200
Subject: ARM: dts: sun7i: Rename sun7i-a20-ir to sun4i-a10-ir
Origin: https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/patch/?id=1715a389949a4a3f03192268dfb3ac5710f2dd3c

The sun7i block is the same as the one in the sun4i, rename the compatible
to reflect this.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>

--

I've already included the matching change to sunxi-cir.c in my pull-req to
Mauro.
Signed-off-by: Maxime Ripard <maxime.ripard@free-electrons.com>

Index: linux-3.16.7-ckt7/arch/arm/boot/dts/sun7i-a20.dtsi
===================================================================
--- linux-3.16.7-ckt7.orig/arch/arm/boot/dts/sun7i-a20.dtsi
+++ linux-3.16.7-ckt7/arch/arm/boot/dts/sun7i-a20.dtsi
@@ -800,7 +800,7 @@
 		};
 
 		ir0: ir@01c21800 {
-			compatible = "allwinner,sun7i-a20-ir";
+			compatible = "allwinner,sun4i-a10-ir";
 			clocks = <&apb0_gates 6>, <&ir0_clk>;
 			clock-names = "apb", "ir";
 			interrupts = <0 5 4>;
@@ -809,7 +809,7 @@
 		};
 
 		ir1: ir@01c21c00 {
-			compatible = "allwinner,sun7i-a20-ir";
+			compatible = "allwinner,sun4i-a10-ir";
 			clocks = <&apb0_gates 7>, <&ir1_clk>;
 			clock-names = "apb", "ir";
 			interrupts = <0 6 4>;
